git-standup: add page (#7584)
tldr-pages/tldr: pages/common/git-standup.md

# git standup

> > See commits from a specified user.

Reviewer: ```suggestion
> See commits from a specified user.
> Part of `git-extras`.
```

# git standup

> See commits from a specified user.
> Part of `git-extras`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-standup>.

> - Show someones commits from the last 10 days:

Reviewer 2: Perhaps this will read better?
```suggestion
- Show a give author's commits from the last 10 days:
```

Author: Do you mean given?

Reviewer: @CleanMachine1 what about this?

`git standup -a {{name|email}} -d {{10}}`

> - Show someones commits from the last 10 days and if they are GPG signed:

Reviewer 2: Perhaps this will read better?
```suggestion
- Show a given author' commits from the last 10 days and whether they are GPG signed:
```
